.gitignore

```diff
@@ -14,3 +14,4 @@ tests/fixtures/drainpipe-test-project/*
 !.yarn/versions
 .npmrc
 lerna-debug.log
+.idea
```

Comment:
```suggestion
```

Reply:
Should I add my IDE settings to the repo?